sass.py

```diff
@@ -694,12 +694,15 @@ def __new__(cls, r, g, b, a):
 SEPARATORS = frozenset((SASS_SEPARATOR_COMMA, SASS_SEPARATOR_SPACE))
 
 
-class SassList(collections.namedtuple('SassList', ('items', 'separator'))):
+class SassList(collections.namedtuple(
+        'SassList', ('items', 'separator', 'bracketed'),
+)):
 
-    def __new__(cls, items, separator):
+    def __new__(cls, items, separator, bracketed):
```

Comment:
I just set it `False` by default.  After a good night's sleep I figured this is a fine default since that used to be the only way.